modules/options.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, pkgs, config, options, ...
```

Comment:
This module should conventionally just be called `default` (and exposed in the flake as `nixosModules.default`).